### PR TITLE
npcap.Npcap version 1.72

### PR DESCRIPTION
**Pull request by aqrh**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93086)

### CODE REVIEW

**Comment by SpecterShell** [comment]
https://github.com/nmap/npcap/issues/588#issuecomment-1079424188

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
+PackageVersion: 1.72
+PackageLocale: en-US
+Publisher: npcap
+PackageName: Npcap
+License: npcap License
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml:9]:
```suggestion
License: Propriety
```

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
+PackageVersion: 1.72
+PackageLocale: en-US
+Publisher: npcap
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml:7]:
```suggestion
Publisher: Nmap Project
```

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Nmap.Npcap
```
Best to keep this as Publisher.Package

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.installer.yaml:4]:
```suggestion
PackageIdentifier: Nmap.Npcap
```
Best to keep this as Publisher.Package

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.yaml:4]:
```suggestion
PackageIdentifier: Nmap.Npcap
```
Best to keep this as Publisher.Package

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
+PackageVersion: 1.72
+PackageLocale: en-US
+Publisher: npcap
+PackageName: Npcap
+License: npcap License
+Copyright: Copyright (c) 2022, Insecure.Com LLC.  All rights reserved.
+ShortDescription: Npcap 1.72 installer
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.locale.en-US.yaml:11]: Please provide a better short description of what Npcap does

File: manifests/n/npcap/Npcap/1.72/npcap.Npcap.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: npcap.Npcap
+PackageVersion: 1.72
```
Comment (Trenly) [inline manifests/n/npcap/Npcap/1.72/npcap.Npcap.installer.yaml:5]:
```suggestion
PackageVersion: 1.72
InstallModes:
- interactive
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Closing due to: No Recent Activity

**Comment by cachius** [comment]
See also https://github.com/nmap/npcap/issues/732

**Pull request closed without merging**